controls/radchartview/features/labels/label-connectors.md

```diff
@@ -28,24 +26,127 @@ The following example demonstrates how to set the LabelConnectorsSettings propet
 				<telerik:PieDataPoint Value="10.35" Label="Holland: 10.35%" />
 				<telerik:PieDataPoint Value="3.55" Label="Luxembourg: 3.55%" />
 			</telerik:PieSeries.DataPoints>
-			
+
 			<telerik:PieSeries.LabelDefinitions>
 				<telerik:ChartSeriesLabelDefinition Margin="-40,0,0,0" />
 			</telerik:PieSeries.LabelDefinitions>
-			
+
 			<telerik:PieSeries.LabelConnectorsSettings>
 				<telerik:ChartSeriesLabelConnectorsSettings />
 			</telerik:PieSeries.LabelConnectorsSettings>
 		</telerik:PieSeries>            
 	</telerik:RadPieChart>
 {{endregion}}
 
-Here is the end result:
+#### __Figure 1: Labels connectors visual appereance__
 ![](images/radchartview-features-smart-labels-labelconnectors_01.png.png)
-	
+
+## Customizing labels connectors points
```

Comment:
I think in the 'labels connectors' phrase one of the words should not be plural. But not sure which one. Can you check this and update it? I think it should be label connectors